src/accounts/account.ts

```diff
@@ -18,14 +18,6 @@ export class Account implements IAccount {
      */
     nonce: INonce = 0;
```

Comment:
this should also be changed to `BigInt`.

Reply:
And at some point, the class should be updated to specs.

Reply:
if it's ok I'll do it in a different pr to keep this for remove only